### DISCUSSION TITLE
Should Graphic Novel be a value in Format

### DISCUSSION THREAD

**Discussion by LokiMidgard** [bookwyrm-social/bookwyrm#3445] [Q&A]
Under Format of a book there is the value Graphic Novel but also Hardcover.

Shouldn't that be something like type? Like Novel/ Graphic Novel/ non-fiction?

After all a Graphic Novel can also be a Hardcover or Paperback.

Maybe I also misunderstand something, since english is not my native tongue and I'm not very familiar with the literature vocabulary.

**Comment by joric3** [comment]
You're right, graphic novel is a type of content, not a publication format. In my opinion it should not be an option on that list.

I think BookWyrm built its data model around books (as in, written words usually by a single author, in long form texts). Users however also try to enter all sorts of stuff which they read - comics, magazines, anthologies...
A sensible data model for comics/graphic novels would require more complexity - not just authors, but artists, inkers, colorists, letterers, and probably editors, too. Not to mention the question of handling individual issues vs. collections like trade paperbacks etc., and how they relate to a series. (This might be related to your question about omnibus editions https://github.com/bookwyrm-social/bookwyrm/discussions/3453: What is a work, what is a multi-part work, how do they relate to each other and their various possible publication forms, and how can we conceptualize and simplify that in a data model.)

**Comment by mouse-reeve** [comment]
I'm not wedded to the format list necessarily, but the reason it's like this is that it's using an external standard: https://schema.org/BookFormatType

**Reply by joric3** [reply]
Thanks! I've brought up my quibbles at https://github.com/schemaorg/schemaorg/issues/3697 - curious to see how the schema.org folks will react. :)
